From: Rohan Garg <rohangarg@ubuntu.com>
Description: Patch to include proper paths to KDE libraries and fix FTBFS issues
Forwarded: Yes, via personal email

Index: fastuserswitch-0.1.1/CMakeLists.txt
===================================================================
--- fastuserswitch-0.1.1.orig/CMakeLists.txt	2010-07-14 17:14:00.000000000 +0530
+++ fastuserswitch-0.1.1/CMakeLists.txt	2010-07-19 21:20:27.000000000 +0530
@@ -4,6 +4,7 @@
 include(KDE4Defaults)
 
 add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
+include_directories(${KDE4_INCLUDES} ${QT_INCLUDES})
 
 set(fastuserswitch_SRCS
     fastuserswitch.cpp
